src/transaction/client.rs

```diff
@@ -263,11 +263,11 @@ impl Client {
     ) -> Result<CleanupLocksResult> {
         debug!(self.logger, "invoking cleanup async commit locks");
         // scan all locks with ts <= safepoint
-        let mut start_key = vec![];
         let ctx = ResolveLocksContext::default();
         let backoff = Backoff::equal_jitter_backoff(100, 10000, 50);
         let req = new_scan_lock_request(
-            mem::take(&mut start_key),
+            options.start_key.clone(),
```

Comment:
Suggest to add `range: impl Into<BoundRange>` argument to `cleanup_locks` instead, to make it consistent in style with other methods.